src/utils/__mocks__/telegramUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { Context } from 'grammy';
+
+/**
+ * Check if a given Context is from a private message.
+ * @param ctx The Telegraf Context to check
+ * @returns True if a private message
+ */
+export const isPrivateChat = (ctx: Context): boolean =>
```

Comment:
It's been a while since I've looked at it, but remember it was being mocked because it was requiring parts of the framework to work correctly. I didn't want to spend a lot of time mocking all of that as well so created a quick mock that did all I needed.